Automattic/liveblog
# Harden Image Check In Media Library

As described in https://github.com/Automattic/liveblog/issues/651, if the media library contains a recently uploaded asset with no thumbnail, the image upload UI breaks completely. This PR resolves the issue by hardening the check to be sure that the image object has sizes defined.

Fixes: https://github.com/Automattic/liveblog/issues/651

### How to test the changes in this Pull Request:

1. Navigate to Media Library, upload any PDF.
2. On `master` branch, view published Liveblog page, and click **Insert Image**.
3. Observe the UI breaks with errors in the Javascript console.
4. Check out this branch, repeat step 3. 
5. Observe the Media Library loads normally.
